src/renderer/pkijs/generatePems/generateRootCA.ts

```diff
@@ -0,0 +1,75 @@
+import { Integer, PrintableString, BitString } from 'asn1js'
+import { Certificate, AttributeTypeAndValue, BasicConstraints, Extension, Time, getCrypto } from 'pkijs'
+
+import { signAlg, hashAlg } from './config'
+import { generateKeyPair, CertFieldsTypes } from './common'
+
+export const createRootCA = async () => {
+  const rootCA = await generateRootCA({
+    commonName: 'Zbay CA',
+    signAlg,
+    hashAlg
+  })
+  // await dumpCertificate(rootCA)
+
+  const rootData = {
+    rootCert: rootCA.certificate.toSchema(true).toBER(false),
+    rootKey: await getCrypto().exportKey('pkcs8', rootCA.privateKey)
+  }
+
+  return {
+    rootCert: Buffer.from(rootData.rootCert).toString('base64'),
+    rootKey: Buffer.from(rootData.rootKey).toString('base64')
+  }
+}
+
+async function generateRootCA({ commonName, signAlg, hashAlg }) {
```

Comment:
in fact we are using same `signAlg` and `hashAlg` throughout entire application.
I think we could simplify a lot if all methods that receive these as parameters get default values from config.ts

Like:
```
async function generateRootCA({ commonName, signAlg = config.signAlg, hashAlg = config.hashAlg }: { commonName: string, signAlg: string, hashAlg: string }): Promise<Certificate> { 
```